.editorconfig

```diff
@@ -0,0 +1,12 @@
+# https://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Great! These will really help us out. 

One thing, this file needs a newline at the end. You can tell because of the red icon.  Otherwise, this is great. Thanks!